### PR TITLE
Security fixes: bump Nokogiri and Puma

### PR DESCRIPTION
**Pull request by laurajaime**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -10,67 +10,81 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (5.2.6)
-      actionpack (= 5.2.6)
+    actioncable (6.0.4.4)
```
Comment (tramuntanal) [inline Gemfile.lock]: Beware I don't think you were upgrading Rails to 6

File: Gemfile.lock [hunk 2]
```diff
@@ -128,88 +143,75 @@ GEM
...
-      decidim-system (= 0.24.3)
-      decidim-templates (= 0.24.3)
-      decidim-verifications (= 0.24.3)
-    decidim-accountability (0.24.3)
-      decidim-comments (= 0.24.3)
-      decidim-core (= 0.24.3)
-      kaminari (~> 1.2, >= 1.2.1)
-      searchlight (~> 4.1)
-    decidim-admin (0.24.3)
+    decidim (0.25.2)
```
Comment (tramuntanal) [inline Gemfile.lock]: Beware I don't think you were upgrading Decidim to the 0.25 series

**Review by tramuntanal — COMMENTED** [review]
Rails and Decidim should not be upgraded

**Comment by laurajaime** [comment]
@tramuntanal can you review it again? I set Decidim version in 0.23.4 because when do `bundle update`, Decidim was updated to 0.25. I need do `bundle update` for fix security updates.

**Review by tramuntanal — APPROVED** [review]
I'd rather prefer `DECIDIM_VERSION = '~> 0.24.3'` so it can be upgraded to latest 0.24.x without having to change this declaration, but let's deploy this right now, 0.24 probably won't have more releases

**Pull request merged**